src/Helpers/RunmeApi.js

```diff
@@ -0,0 +1,61 @@
+import { removeDoubleSlashes } from './Const'
+import { RUNME_API } from '../env'
+
+function setApiUrl (path, protocol = 'http') {
+  const secure = RUNME_API.secure ? 's' : ''
+  const url = `${protocol}${secure}://${RUNME_API.host}/${path}`
+
+  return removeDoubleSlashes(url)
+}
+
+export function wsBuild (buildId, callback) {
+  const url = setApiUrl('ws', 'ws')
+  const ws = new WebSocket(url)
```

Comment:
This websocket is never closed, if the user leaves the route and come back would this be ran again? I know that your PR didn't change the original behaviour, just raising the concern 😁

Reply:
Good one, will check/fix that in a new PR 😊